src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -482,11 +470,17 @@ public Rotation2d getRotation() {
 
   /** Sets the current odometry pose. */
   public void setPose(Pose2d pose) {
-    SwerveSubsystem.pose = pose;
+    this.pose = pose;
     try {
       estimator.resetPosition(pose.getRotation(), lastModulePositions, pose);
     } catch (Exception e) {
     }
```

Comment:
why does this need a try catch?

Reply:
bc of this https://github.com/HighlanderRobotics/Crescendo/pull/25#discussion_r1467247571

Reply:
me when i have the memory of a goldfish